src/session/SessionMiddleware.ts

```diff
@@ -28,13 +29,10 @@ function sessionRequestHandler(config: CookieConfig, sessionStore: SessionStore)
 
         if (sessionCookie) {
 
-            console.log("Got a session cookie.");
-            console.log(`REQUEST: ${request.url}`);
-            console.log(`COOKIE: ${sessionCookie}`);
+            loggerInstance().info(`Session cookie - REQUEST: ${request.url} - COOKIE: ${sessionCookie}`);
```

Comment:
I like the fact that you combined multiple lines into single log entries, it is right thing to do, especially important when logs are being aggregated.

I do not quite like logged message e.g. 'Session cookie'? That triggered question 'Session cookie what?' in my head. I started to think what it really means and what it signals. I suggest you make it bit more meaningful describing what happened.

Reply:
I would generally want to see the following logs:
1. [INFO] `Session cookie ${cookieID} found in request: ${request.url}`
2. [INFO] `Session cookie ${cookieID} not found in request ${request.url}`
3. [DEBUG] `Session successfully loaded from cookie ${cookieID}`
4. [ERROR] `Session loading failed  from cookie ${cookieID} due to error: ${err}`